### ISSUE TITLE
Failing to build benchmark

### ISSUE THREAD

**Issue by cswinter** [facebookincubator/gloo#180]
I'm trying to build the benchmarks on latest master. Compilation seemingly succeeds, but does not produce a benchmark binary:

```
root@managed-worker-3jq5:/gloo# mkdir build
root@managed-worker-3jq5:/gloo# cd build
root@managed-worker-3jq5:/gloo/build# cmake ../ -DBUILD_BENCHMARK=1
-- The CXX compiler identification is GNU 7.4.0
-- The C compiler identification is GNU 7.4.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Build type not set -- defaulting to Release
-- Configuring done
-- Generating done
-- Build files have been written to: /gloo/build
root@managed-worker-3jq5:/gloo/build# ls -l gloo/benchmark/benchmark
ls: cannot access 'gloo/benchmark/benchmark': No such file or directory
root@managed-worker-3jq5:/gloo/build# ls gloo/benchmark
CMakeFiles  Makefile  cmake_install.cmake
root@managed-worker-3jq5:/gloo/build# git rev-parse HEAD
5a0e5f6f7714bdcf0c5c74c92f3594063906060b
```

**Comment by pietern** [comment]
You're not running `make` to compile everything.

This was missing from the build instructions and is being fixed in #202.

**Comment by pietern** [comment]
Thanks for posting the issue.
